lib/vm/src/instance.rs

```diff
@@ -278,8 +278,8 @@ impl Instance {
     }
 
     /// Return a raw pointer to the vmctx used by compiled wasm code.
-    pub fn vmctx_ptr(&self) -> *const VMContext {
-        self.vmctx() as *const VMContext
+    pub fn vmctx_ptr(&self) -> *mut VMContext {
```

Comment:
Where's the code that needs this to be mutable?

Reply:
It's like that on master

https://github.com/wasmerio/wasmer/blob/master/lib/vm/src/instance.rs#L281-L283

I probably looked at master and got confused or something 🤷

Reply:
Actually I don't even know how you commented on that, I don't see that in the diff 😆